gh2/csv.py

```diff
@@ -207,24 +209,26 @@ def set_headers(repo, labels=None):
     return headers
 
 
-def main():
-    parser = make_parser()
+def get_all_labels(repositories):
```

Comment:
This function has some unnecessary iteration and is duplicating functionality provided by `set`, I think something like the following would be better:
```python
def get_all_labels(repositories):                                                 
    labels = set()                                                                 
    for repo in repositories:                                                      
        labels.update(l.name for l in repo.labels())                               
    return sorted(labels)
```

Note, this change would require line 208 being updated to:
```python
        headers.extend('Label: ' + label for label in labels)
```